estate/models/offer.py

```diff
@@ -0,0 +1,41 @@
+from odoo import api, fields, models
+
+class Offer(models.Model):
+        _name = "estate.property.offer"
+        _description = "Offers"
+
+        price = fields.Float(required=True)
+        buyer_id = fields.Many2one("res.partner", string="Partner")
+        validity = fields.Integer(default=7, required=True, string="Validity (days)")
+        deadline = fields.Date(compute="_compute_deadline", inverse="_inverse_deadline", string="Deadline Date")
+        status = fields.Selection(
+                selection=[
+                    ("new", "New"),
+                    ("accepted", "Accepted"),
+                    ("refused", "Refused"),
+                ],
+                default="new",
+                copy=False,
+                required=True,
+        )
+
+        property_id = fields.Many2one("estate.property", required=True)
+
+        @api.depends("validity", "create_date")
+        def _compute_deadline(self):
+            for estate in self:
```

Comment:
It would more appropriate to call your record with the model name
```suggestion
            for offer in self:
```